frontend/src/tests/utils.js

```diff
@@ -160,6 +160,7 @@ export { expect, test, it, describe, beforeAll };
 
 /** URL prefix for making API calls from inside a docker image */
 export const API_URL = "http://backend:3000/api/v1";
```

Comment:
Couldn't find why `API_URL` is exported, it seems to be not used outside of this file?